packages/gatsby/src/commands/develop-process.ts

```diff
@@ -80,7 +80,7 @@ const openDebuggerPort = (debugInfo: IDebugInfo): void => {
 }
 
 module.exports = async (program: IDevelopArgs): Promise<void> => {
-  reporter.setVerbose(program.verbose)
+  reporter.setVerbose(isTruthy(process.env.GC_VERBOSE) || program.verbose)
```

Comment:
Why GC_VERBOSE, not GC_VERBOSE?

Reply:
Good point! Switching to `VERBOSE`

Reply:
Sorry for adding more to it - should it be `GATSBY_VERBOSE` as we prefix bunch of other things with `GATSBY_`? (feel free to ignore of course)

Reply:
@pieh Those are included in client side code though 😓